Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml

```diff
@@ -390,6 +390,36 @@
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHelmetHardsuitSecurityRed
 
+#Quartermaster's Hardsuit
+- type: entity
+  parent: ClothingOuterHardsuitBase
+  id: ClothingOuterHardsuitQm #DO NOT MAP - https://github.com/space-wizards/space-station-14/pull/19738#issuecomment-1703486738
+  name: luxury mining hardsuit
+  description: A refurbished mining hardsuit, fashioned after the Quartermaster. Graphene lining provides less protection, but is much easier to move.
```

Comment:
You need to remove references (ID, comment, description) to this being related to the QM. Ideally the sprite would be renamed too.

Reply:
I was told leaving description references to the Quartermaster would be fine. I can definitely change the internal thongs for sure, but is it seriously imperative to remove every vein and spine of intent I had with the original PR?

Reply:
Just looked through the relevant discord discussion. Description is fine, yeah.